### DISCUSSION TITLE
ZZT BMS058

### DISCUSSION THREAD

**Discussion by Intrerio** [Louisvdw/dbus-serialbattery#443] [Q&A] [answered]
Hi friends. I bought batteries https://www.fiberx.shop/wp-content/uploads/2021/04/teh.spec_.-ZTT48100.pdf and can somebody help me with info about how I can connect this to Victron Quattro II 5000?

**Comment by MisterX1000** [comment]
Hey, you will find a lot of information here:

https://www.victronenergy.com/live/ess:quick-installation-guide

**I would strongly suggest that you should read this manual here:**
https://www.victronenergy.com/upload/documents/Energy_Storage_System/ESS_design_and_installation_manual-en.pdf

Additional and very helpful information can be found on youtube when you search for "Victron ESS"

Best regards
Alex

**Reply by Intrerio** [reply]
I'm already connected this batteries to Victron and now they are working but I can't connect batteries BMS to Victron. I bought USB to RS485 adapter, made cable for it but I don't know what I need to configure to make this devices working well :-(. I was try to read manuals and forums but all what I found it's Louis git and here I found info that I need to use some special driver but here is not driver for my BMS.Maybe somebody can help me with it? I don't know what to do cause I live in Ukraine and I can't find any help cause war destroy any ways to find help

**Reply by MisterX1000** [reply]
When you BMS is not listed in Louis BMS list you could open an issue ask for implementation of your BMS in Louis driver. 

https://github.com/Louisvdw/dbus-serialbattery/wiki/BMS-types-supported

But in principle your ESS works also without the "serialbattery" driver. 
In this case the Victron is the master of the voltages and currents :-) You just need to set it up in "VE configure" like discribed in the links in my post.
I hope I could help you. Best wishes to you

**Reply by Intrerio** [reply, marked as answer]
yes, my batteries is not listed there. I open new issue and added communication protocol of my BMS, maybe Louis will help me with my problem. Thank you for your help. I also configured all in ve.configure but I have some alarm from time to time cause BMS is not working with inverter.
